src/App.css

```diff
@@ -57,3 +58,11 @@
     grid-template-columns: repeat(4, 1fr);
   }
 }
+
+.modal__component {
```

Comment:
Why is there a modal styling in the app.css?

Reply:
is there because I styled the button(Skills) from the App.js but isn't the right name for it. I will change the name of the class.